pkg/manager/watcher_test.go

```diff
@@ -77,8 +77,8 @@ func TestParseNewBgpAnnotations(t *testing.T) {
 	node.Annotations = map[string]string{
 		"bgp/bgp-peers-0-node-asn": "65000",
 		"bgp/bgp-peers-0-peer-asn": "64000",
-		"bgp/bgp-peers-0-peer-ip":  "10.0.0.254",
-		"bgp/bgp-peers-0-src-ip":   "10.0.0.1,10.0.0.2,10.0.0.3",
+		"bgp/bgp-peers-0-peer-ip":  "10.0.0.1,10.0.0.2,10.0.0.3",
+		"bgp/bgp-peers-0-src-ip":   "10.0.0.254",
```

Comment:
If I'm not mistaken, these two expected values were swapped, the code that generates the peers looks correct, it was just the wrong expectation